qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/VisitNotePage.java

```diff
@@ -87,6 +89,15 @@ public void enterSecondaryDiagnosis(String diag) {
 		clickOn(UI_MENU_ITEM);
 	}
 
+	public void removeDiagnosis() {
+		clickOn(REMOVE_DIAGNOSIS_BUTTON);
```

Comment:
@Muta-Jonathan can we see through the idention here

Reply:
I think its ok @Ndacyayisenga-droid